### ISSUE TITLE
Ability to create own tier groups

### ISSUE THREAD

**Issue by NDR-113** [deckerst/aves#538]
**Is your feature request related to a problem? Please describe.**
Aves has this feature of "Tiers". The ones i've came across being "Pinned", "Downloads", "Common" and "Others". This is useful and perhaps the functionality could be expanded, since it's a great idea. I have several albums, and right now the only way I "group" them is by a suffix in their name, so they appear together in alphabetic sorting in collections. For example, screenshots of book passages go to B_(book name).  Then to further order i can append a numbering system 1_, 2_ and so on.

**Describe the solution you'd like**
The ability to 1- create/name and 2- position our own tier groups.
1. With this I mean create other tier groups beyond the 4 existing ones, where we can place albums inside. If there was button to collapse  / expand even better.
2. Just through long press, drag and drop. Although just 1) functionality would already be a big improvement by itself and we would still be able to order with prefix on the tier group name.

**Comment by deckerst** [comment]
For completeness, at the moment the tiers are these:
```
  "albumTierNew": "New",
  "albumTierPinned": "Pinned",
  "albumTierSpecial": "Common",
  "albumTierApps": "Apps",
  "albumTierVaults": "Vaults",
  "albumTierRegular": "Others",
```

I guess "Downloads" is a mistranslation of `Apps` in your locale.

Back to your point, I see the value of your proposal. There are other approaches, though, like the nested albums:
- #268
 
The original idea with tiers is that they're derived automatically from the collection and sorted in a (hopefully) sensible way. All the ideas for custom sort, custom grouping, custom tiers are interesting but come after ideas that improve things by default for everybody.

**Comment by DroidFloyd** [comment]
I agree with NDR's #1 tiers option. Just realized it would be quite similar to how Aves Settings is setup with drop-down menus, which provides a clean, organized user interface.

**Comment by danudesigns** [comment]
Hello :) I believe I've encountered the same limitation that I think would be conveniently solved by *tiered groups?*. 
I've attached a screenshot of something of what I'm imagining. 

It would be nice to categorise, order & rename the album groups. Collapse and expand would be a very neat feature too. 
Drag and reorder (even between group categories ) could be a nice touch too! 

(I know I've drawn a direct "create new album too... But tbh that's not as much a priority as it can be done by batch grouping... The ui priority imo is the custom categorisation of albums.) 

Hope this what the OP was referring to because I sure would appreciate this update! 

![Screenshot 2025-10-04 205627.png](https://github.com/user-attachments/assets/02efa14f-e2a7-44f9-b3c7-96526a19f0a9)

**Comment by danudesigns** [comment]
The explained application for the above layered categorisation of Grouped Albums is as such: 

The *Hobbies* nest could have multiple album groups which in turn would have subcategories(albums) ... 
Gardening, > indoor plants, gardening tips,... etc etc
Painting, > so on ...
And so on... 

I hope I've justified the practical application for my rationale :)

**Comment by NDR-113** [comment]
Hi, I'm glad to see more people finding the idea to be useful to be implemented. I believe the author is halfway there already with the recent implementation of Groups. They work nicely and it has made organizing my albums already much better than it was. I personally am ok without the collapsing feature (and actually prefer not to have it) but I understand some people might like it. The main things I would say are currently missing:

1- Being able to create other groups of albums other than "Groups"
2- Being able to order the albums as you want, also inside groups

**Comment by deckerst** [comment]
> Being able to create other groups of albums other than "Groups"

I don't understand.

**Comment by danudesigns** [comment]
Hi and thanks André for the clearer summary :') . 

The groups of groups/ nested groups/.. 
Is what I've placeholdered as 'hobbies, travel,' .. Common, Apps...in my screenshot above. 

The feature I was also trying to pinpoint is the ability to create a new "nest/groupofgroups .."

**Comment by deckerst** [comment]
It's already possible to create/reorganize groups in groups.

**Comment by danudesigns** [comment]
Sorry if I'm confusing the terms..  
Maybe I should have said nests of groups. 

So there's Apps, collating albums deriving from apps. 

Then there's Common collating common albums, 

Then there's 'Groups' which collates sets of grouped albums. 

*The new feature* would be the ability to add multiple renameable new 'Groups' which could collate a different set of grouped albums, and for each '''Groups of groups''' to be editable and renamed. 

At the moment, all grouped albumss fall under the same "nest/category" (i.e. the single general 'Groups') which goes alongside, 'Common', 'Apps'...

**Comment by NDR-113** [comment]
@deckerst 

1. Select two albums
2. Change grouping
3. Then you can chose either an existing group or create a new one.
4. In either case, the album will be placed inside the "albumTier" named "Group"

My suggestion (and i think from what i understand danu is saying similar) is to be able to create (or better yet) edit these albumTiers. I personally don't need a "Common" albumTier for example. This would basically allow one level more of hierarchical organization.

Also when you sort albums you have to chose "by name", "by date", etc. There's no way to sort as in a drag and drop, or a custom way. Same sorting idea to be applicable for albumTiers. "Pinned" show at the top (makes sense). But then comes "Groups", then "Common", etc. There's no way to change this sorting afaik.

**Comment by danudesigns** [comment]
Honestly the drag and drop is a nice touch, but not as big of a priority from my suggestion as much as the framework of "Group-inception" 😅. 

Thanks André, Yes, "changing grouping" > "create new group" or "select from existing" would be a solved path. 

(Aha.. album tier is the correct term)
